Gordon360/Services/ProfileService.cs

```diff
@@ -65,6 +65,29 @@ public AlumniProfileViewModel GetAlumniProfileByUsername(string username)
                 result = alumni;
             return result;
         }
+
+        /// <summary>
+        /// get mailbox combination
+        /// </summary>
+        /// <param name="username">The current user's username</param>
+        /// <returns>MailboxViewModel with the combination</returns>
+        public MailboxViewModel GetMailboxCombination(string username)
+        {
+            var mailboxNumber = 
+                Data.StudentData
+                .FirstOrDefault(x => x.AD_Username.ToLower() == username.ToLower())
+                .Mail_Location;
+
+            MailboxViewModel info = new MailboxViewModel();
+
+            if (mailboxNumber != null)
```

Comment:
Cases to consider for safety:

1: What if called for a non-student, so have mail stop instead of mailbox?  They won't be found in the StudentData, so will get the default answer.

2: What if a student doesn't have a mailbox number?  null check will give the default answer for no mail location at all (including grad students, who have no mail location at all).  Looking in the database, it appears there are no non-numeric mail locations for students.

3: What if the mailbox number is invalid?  We think it will throw an exception, which will serve a 500.  That's awkward, but we don't think it will happen.  The ideal might be to throw a custom exception with a message about what wasn't found.  But since we don't have a test case, or a good way to create one, it will be hard to get this right.  

I think it's OK to commit this as-is.